yt_dlp/extractor/bilibili.py

```diff
@@ -1929,10 +1921,12 @@ def _extract_video_metadata(self, url, video_id, season_id):
 
         # XXX: webpage metadata may not accurate, it just used to not crash when video_data not found
         return merge_dicts(
-            self._parse_video_metadata(video_data), self._search_json_ld(webpage, video_id, fatal=False), {
-                'title': self._html_search_meta('og:title', webpage),
-                'description': self._html_search_meta('og:description', webpage)
-            })
+            self._parse_video_metadata(video_data), {
+                'title': get_element_by_class(
+                    'bstar-meta__title', webpage) or self._html_search_meta('og:title', webpage),
+                'description': get_element_by_class(
+                    'bstar-meta__desc', webpage) or self._html_search_meta('og:description'),
+            } or self._search_json_ld(webpage, video_id, fatal=False))
```

Comment:
maybe always merge the json ld but default to dict to prevent unnecessary warnings in every log
```suggestion
            }, self._search_json_ld(webpage, video_id, default={}))
```